sdk/eventhub/Azure.Messaging.EventHubs/src/EventHubsException.cs

```diff
@@ -124,20 +124,8 @@ public override string Message
         ///
         public EventHubsException(string eventHubName,
                                   string message,
-                                  FailureReason reason) : this(default, eventHubName, message, reason, null)
+                                  FailureReason reason) : this(eventHubName, message, reason, null)
```

Comment:
Not resolvable: The ordering of these parameters throws me off.. I figured it would always be Mandatory parameters first, and then optional ones like `bool isTransient`. Threw me for a tizzy when I saw `this(default,...)` 😆

Reply:
I'm not going to lie....  this type came in very, very hot before our initial GA.   I think we messed a few things up in here with the overloads and ordering and such.

Reply:
though, I honestly don't know why `default` was used there instead of `false`.    I should probably fix that.